src/main/java/frc/robot/RobotMap.java

```diff
@@ -13,10 +14,16 @@
 public class RobotMap {
 
     private static RobotMap robotMap;
-    public static final WPI_TalonSRX leftFrontTalon = new WPI_TalonSRX(0);
-    public static final WPI_TalonSRX leftRearTalon = new WPI_TalonSRX(1);
-    public static final WPI_TalonSRX rightFrontTalon = new WPI_TalonSRX(2);
-    public static final WPI_TalonSRX rightRearTalon = new WPI_TalonSRX(3);
+    public static final WPI_VictorSPX leftFrontTalon = new WPI_VictorSPX(0);
+    public static final WPI_VictorSPX leftRearTalon = new WPI_VictorSPX(1);
+    public static final WPI_VictorSPX leftMiddleTalon = new WPI_VictorSPX(2);
+    public static final WPI_VictorSPX rightFrontTalon = new WPI_VictorSPX(3);
+    public static final WPI_VictorSPX rightRearTalon = new WPI_VictorSPX(4);
+    public static final WPI_VictorSPX rightMiddleTalon = new WPI_VictorSPX(5);

```

Comment:
Motory jsou pojmenované front, center a right, bylo by tedy fajn to tak mít i v kódu.